lib/core/src/lib/login/components/login-dialog.component.html

```diff
@@ -1,25 +1,23 @@
-<header
-    mat-dialog-title
-    data-automation-id="login-dialog-title">{{data?.title}}
+<header mat-dialog-title data-automation-id="login-dialog-title">
```

Comment:
In my opinion, the styling that was before is better for readability

Reply:
fixed in new commit